modules/tensor_mechanics/src/utils/EulerAngles.C

```diff
@@ -15,3 +14,11 @@ EulerAngles::random()
   Phi  = std::acos(1.0 - 2.0 * MooseRandom::rand()) / libMesh::pi * 180.0;
   phi2 = MooseRandom::rand() * 360;
 }
+
+void
+EulerAngles::random(MooseRandom & random)
+{
+  phi1 = random.rand() * 360.0;
```

Comment:
Oops! You've accidentally called the static methods here. Make sure you pass the index for which state you want to manipulate (zero is fine here).